src/Symfony/Component/Security/Core/composer.json

```diff
@@ -17,6 +17,7 @@
     ],
     "require": {
         "php": "^5.5.9|>=7.0.8",
+        "psr/container": "^1.0",
```

Comment:
Shouldn't this be in require-dev? The dep is required only when using `new Security()` (same below).